config/routes.js

```diff
@@ -29,7 +28,9 @@ export const HomeStack = StackNavigator({
   Login: {
     screen: LoginScreen,
     navigationOptions: {
-      headerTitle: 'Login'
+      headerTitle: 'Login',
+      headerLeft: null,
```

Comment:
nit: is headerleft necessary?

Reply:
Yeah! b/c it defaults to the back button to the previous screen rip